code/modules/VR/vr_sleeper.dm

```diff
@@ -177,6 +177,8 @@
 		vr_human.undershirt = H.undershirt
 		vr_human.underwear = H.underwear
 		vr_human.updateappearance(TRUE, TRUE, TRUE)
+		if(!is_synth(vr_human) || !ishumanbasic(vr_human))
```

Comment:
this would transform everyone, including synths and humans
a synth isn't a human
and a human isn't a synth